docs/releases/upgrading/upgrade-v11.0.md

```diff
@@ -0,0 +1,68 @@
+# Upgrading from v10 to v11
```

Comment:
I'm not sure where/how we can make it simple for the user but they need to update their python version:

For local installations:

- Destroy venv
- Recreate + reinstall

For containers:

- Update the base image + rebuild (what about Pipenv lock? the source needs to upgrade to py 3.9 no?)

Reply:
added.